mediator/med_methods_mod.F90

```diff
@@ -71,6 +77,8 @@ module med_methods_mod
 #ifdef DIAGNOSE
   private med_methods_Array_diagnose
 #endif
+  private med_methods_check_for_nans
```

Comment:
It is nice to have this function to capture nans when they show up! Good tool for debugging

Reply:
Yes, I haven't enabled it in any test (added a new export variable, define it false, add it to nems.configure files), but I could. We could enable it for just the debug test. 

Initially, this feature appeared to have a big impact on run time. But it turned out to be the change in the pio default rearranger.